src/software/new_geom/point_boost_geometry_compatability.h

```diff
@@ -0,0 +1,44 @@
+/**
+ * This file contains everything neccesarry to allow our `point` type to be used with
+ * boost algorithms. Because it includes parts of boost geometry, it is very expensive,
```

Comment:
specify it's expensive with respect to build time

Reply:
added